api/s2n.h

```diff
@@ -83,7 +83,14 @@ struct s2n_connection;
  * with S2N.
  */
 S2N_API
-extern void s2n_crypto_disable_init(void);
+extern int s2n_crypto_disable_init(void);
```

Comment:
_Technically_ this is a breaking change but hopefully no one has consumed it yet :smile:

Reply:
Yeah, I'm counting on that. The good news is, even if they consumed it, this cannot cause a compile failure.

Reply:
It still could, but unlikely: https://godbolt.org/z/zPeh99793